apps/api/src/schema/websoc.ts

```diff
@@ -75,18 +75,37 @@ const isValidRestrictionCode = (code: string): code is (typeof restrictionCodes)
 export const websocQuerySchema = z.object({
   year: z
     .string({ message: "Parameter 'year' is required " })
-    .length(4, { message: "Parameter 'year' must have length 4" }),
-  quarter: z.enum(terms),
+    .length(4, { message: "Parameter 'year' must have length 4" })
+    .openapi({ description: "The academic year for course search", example: "2025" }),
+  quarter: z.enum(terms).openapi({
+    description:
+      "The academic quarter 'Fall', 'Winter', 'Spring', 'Summer1', 'Summer10wk', or 'Summer2'",
```

Comment:
Giving a description for enum terms which lists variants is redundant since Scalar already does so. In general, make sure you're checking the auto-generated `/reference` to see what your changes look like.